OneMore/Commands/Snippets/InsertDateTimeCommand.cs

```diff
@@ -0,0 +1,68 @@
+//************************************************************************************************

```

Comment:
What I would prefer to do is enhance the InsertDateCommand. That follows the OneMore pattern of other commands that combine very similar functionality.

Currently, InsertDateCommand is bound to Ctrl+Shift+D to insert yyyy-MM-dd. We could also bind Ctrl+Alt+Shift+D to this command which would emit "yyyy-MM-dd hh:mm tt" as you suggest.

I think this would be a cleaner approach and avoid duplicate a bit of code.

Reply:
Hotkeys are registered declaratively in [AddinCommands.cs](https://github.com/stevencohn/OneMore/blob/4f993a517ec6f2e1250efdfe16f749677488a5b2/OneMore/AddInCommands.cs#L256).

But it would be easy enough to add a secondary binding in the bottom of [AddInHotkeys.cs](https://github.com/stevencohn/OneMore/blob/4f993a517ec6f2e1250efdfe16f749677488a5b2/OneMore/AddInHotkeys.cs#L62) by calling  `HotkeyManager.RegisterHotKey(action, hotkey);` directly.

Reply:
The second binding should pass a boolean argument indicating the request to include the time in the output